README.md

```diff
@@ -50,12 +59,12 @@ function configureBot(bot) {
     bot.setDebug(true);
 
     // announce in chat when Bot spawns
-    bot.on('spawn', function() {
+    bot.mineflayer().on('spawn', function() {
```

Comment:
With your change to on, we don't need `mineflayer()` in these examples right?

Reply:
correct, got the one at the bottom of the file, missed this one

Reply:
resolved